.gitignore

```diff
@@ -1,9 +1,12 @@
 sqlScripts/
 *.iml
 .idea/
+.log/
 
 target/
 
 src/main/resources/liquibase.properties
 
-dependency-reduced-pom.xml
\ No newline at end of file
+dependency-reduced-pom.xml
+
+locations.json  
```

Comment:
It looks like locations.json is still checked in, even though it's (correctly) in .gitignore.